File: snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/Production.kt
package com.example.kotlinmpplib

import breez_sdk.*
class Production {
    fun productionNodeConfig(): NodeConfig {
        // ANCHOR: moving-to-production
        // Read in your Greenlight credentials from the file 
        // system, environment variable or build config
        val developerKey = emptyList<UByte>()
        val developerCert = emptyList<UByte>()
        val greenlightCredentials = GreenlightCredentials(developerKey, developerCert)

        val nodeConfig = NodeConfig.Greenlight(GreenlightNodeConfig(greenlightCredentials, null))
        // ANCHOR_END: moving-to-production
        return nodeConfig
    }
}

File: snippets/react-native/production.ts
import {
  type GreenlightCredentials,
  type NodeConfig,
  NodeConfigVariant
} from '@breeztech/react-native-breez-sdk'

const productionNodeConfig = (): NodeConfig => {
  // ANCHOR: moving-to-production
  // Read in your Greenlight credentials from the file
  // system, environment variable or build config
  const developerKey: number[] = []
  const developerCert: number[] = []
  const greenlightCredentials: GreenlightCredentials = {
    developerKey,
    developerCert
  }

  const nodeConfig: NodeConfig = {
    type: NodeConfigVariant.GREENLIGHT,
    config: {
      partnerCredentials: greenlightCredentials
    }
  }
  // ANCHOR_END: moving-to-production
  return nodeConfig
}
